Library/Homebrew/requirements/x11_requirement.rb

```diff
@@ -27,20 +27,16 @@ def min_xdpyinfo_version
 
   satisfy build_env: false do
     if which_xorg = which("Xorg")
-      version = Utils.popen_read which_xorg, "-version", err: :out
-      next false unless $CHILD_STATUS.success?
-      version = version[/X Server (\d+\.\d+\.\d+)/, 1]
-      next false unless version
-      Version.new(version) >= min_version
-    elsif which_xdpyinfo = which("xdpyinfo")
-      version = Utils.popen_read which_xdpyinfo, "-version"
-      next false unless $CHILD_STATUS.success?
-      version = version[/^xdpyinfo (\d+\.\d+\.\d+)/, 1]
-      next false unless version
-      Version.new(version) >= min_xdpyinfo_version
-    else
-      false
+      version = Utils.popen_read(which_xorg, "-version", err: :out)[/X Server (\d+\.\d+\.\d+)/, 1]
```

Comment:
If this doesn't return a string then the `[/X Server` will fail. It'd be better to keep the version separate like before with `version = version[/X Server (\d+\.\d+\.\d+)/, 1]`

Reply:
`Utils.popen_read` always returns a string, even when it fails.

```
❯❯❯ brew irb <<<'Utils.popen_read("foo")'
Switch to inspect mode.
Utils.popen_read("foo")
""
```

Reply:
Cool 👍